distage/distage-framework/src/main/scala/izumi/distage/effect/modules/CatsDIEffectModule.scala

```diff
@@ -39,12 +17,10 @@ trait CatsDIEffectModule extends ModuleDef {
   make[ContextShift[IO]].from((_: PublicIOApp).contextShift)
   make[Timer[IO]].from((_: PublicIOApp).timer)
 
-  make[PublicIOApp].from {
-    new PublicIOApp { override def run(args: List[String]): IO[ExitCode] = IO.pure(ExitCode(0)) }
-  }
+  implicit val cs: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
```

Comment:
@VladPodilnyk ContextShift is already bound in DI, why bind it at top-level? (And also this uses a different ContextShift not from DI, which results in having multiple ContextShift's bound to different thread pools in the app)